src/canadiantracker/scraper.py

```diff
@@ -159,8 +159,9 @@ def scrape_skus(db_path: str, products: str | None):
         db_path
     )
 
+    products_list: list[str] | None = None
```

Comment:
For fun, I tried to see if this would work, redefining the variable with a new type:

    products: list[str] | None = products.split(",") if products is not None else None

But PyLance now complains:

    Parameter declaration "products" is obscured by a declaration of the same name

So using a new variable is fine with me, and it's also very clear.